src/pages/Compare/index.spec.tsx

```diff
@@ -128,6 +128,23 @@ describe(__filename, () => {
     expect(root.find(VersionChooser)).toHaveProp('addonId', addonId);
   });
 
+  it('renders a loading message when no diff has been loaded', () => {
+    const addonId = 123;
+    const baseVersionId = 1;
```

Comment:
I'm not sure why you're specifying a `baseVersionId` here. The only place you use it is to set the `id` when creating `version`, and then later you just use `version.id`, so it seems like you might as well just remove `baseVersionId`, and not bother overriding `id` for the version.

I.e., this looks like the same test:

```diff
diff --git a/src/pages/Compare/index.spec.tsx b/src/pages/Compare/index.spec.tsx
index b3111d9..b7cfddd 100644
--- a/src/pages/Compare/index.spec.tsx
+++ b/src/pages/Compare/index.spec.tsx
@@ -129,14 +129,11 @@ describe(__filename, () => {
   });
 
   it('renders a loading message when no diff has been loaded', () => {
-    const addonId = 123;
-    const baseVersionId = 1;
-    const version = { ...fakeVersionWithDiff, id: baseVersionId + 1 };
+    const version = fakeVersionWithDiff;
     const store = configureStore();
     store.dispatch(versionsActions.loadVersionInfo({ version }));
 
     const root = render({
-      addonId: String(addonId),
       headVersionId: String(version.id),
       store,
     });
```